backup-stores/gcs/pom.xml

```diff
@@ -21,6 +21,7 @@
 
   <name>Zeebe Backup Store for GCS</name>
 
+<<<<<<< HEAD
```

Comment:
@oleschoenburg I think this is not expected :D

Reply:
Yikes :+1:

Reply:
That's what I get from trying to merge without a merge tool.